source/Halibut/Transport/Streams/NetworkTimeoutStream.cs

```diff
@@ -30,7 +30,7 @@ public override async ValueTask DisposeAsync()
 
         public override async Task FlushAsync(CancellationToken cancellationToken)
         {
-            ThrowIfAlreadyTimedOut();
+            ThrowIfAlreadyCancelledOrTimedOut();
```

Comment:
The method and properties were a slight lie as it can be timed out but it can also be cancelled